spec/support/fixture.rb

```diff
@@ -31,5 +31,13 @@ def read_file(type, fixture)
       File.read path
     end
 
+    def nori
+      nori_options = {
```

Comment:
We should not evaluate nori_options every time you call this method.  You are creating objects you don't need, like a lambda

Move this into a method nori_options and call it here.  Look how simple this method gets.  And we only call nori_options as needed.
```
def nor
  @nori ||= Nori.new(nori_options)
end
```